config-templates/redhat/odf-local/changelog.json

```diff
@@ -31,9 +30,9 @@
         "4.14": {
             "status": "supported",
             "latest-revision": {
-                "revision": "5",
-                "release-date": "05 June 2024",
-                "cves": "CVE-2024-33600, CVE-2024-33601, CVE-2023-7008, CVE-2023-6004, CVE-2024-26458, CVE-2024-26461, CVE-2024-2961, CVE-2023-6918, CVE-2024-33599, CVE-2024-33602",
+                "revision": "6",
+                "release-date": "21 June 2024",
+                "cves": "CVE-2023-45288",
                 "golang": "1.21.10",
```

Comment:
Please confirm golang from travis. I think it should be 1.21.11 most probably. Please confirm.

Reply:
done, not applicable for 4.12 ODF and 4.13 fix is not available as it is a dependent with RH repos

Reply:
So customer using 4.12 & 4.13 will continue to have security issues in their environment?